moped-editor/src/components/FileUpload/FileUpload.js

```diff
@@ -274,13 +270,7 @@ const FileUpload = props => {
         }}
         beforeAddFile={handleBeforeAdd}
         onprocessfile={(error, file) => handleFileAdded(error, file)}
-        onupdatefiles={setFiles}
-      >
-        {/* Update current files  */}
-        {files.map(file => (
-          <File key={file} src={file} origin="local" />
```

Comment:
this is showing an error in the console when running locally, 
```
react_devtools_backend.js:2655 Warning: React.jsx: type is invalid -- expected a string (for built-in components) or a class/function (for composite components) but got: undefined. You likely forgot to export your component from the file it's defined in, or you might have mixed up default and named imports.

Check your code at FileUpload.js:283.
```

files is an array of filepond file objects, not an array of our files. when I commented out this section, I saw no difference in the UI. I think this is supposed to show the files that are staged for upload as you add them to the dialog. But since we only allow users to upload one file at a time, the upload status that shows up represents the file that is added. 

tldr: this isnt doing anything but throwing an error so i removed it.